stories/internal/Export.tsx

```diff
@@ -0,0 +1,225 @@
+                        onExportReady={onExportReady}
+                        onError={onErrorHandler}
+                        LoadingComponent={null}
+                        ErrorComponent={null}
+                    />
+                )}
+            </WrapperComponent>
+        </div>
+    ))
+    .add('export table data', () => (
+        <div style={WRAPPER_STYLE}>
+            <WrapperComponent>
+                {(onExportReady: OnExportReady) => (
+                    <Table
+                        projectId="storybook"
+                        measures={[MEASURE_1, MEASURE_2]}
+                        attributes={[ATTRIBUTE_1]}
+                        onExportReady={onExportReady}
+                        onError={onErrorHandler}
+                        LoadingComponent={null}
+                        ErrorComponent={null}
+                    />
+                )}
+            </WrapperComponent>
+        </div>
+    ))
+    .add('export pivot table data', () => (
+        <div style={WRAPPER_STYLE}>
+            <WrapperComponent>
+                {(onExportReady: OnExportReady) => (
+                    <PivotTable
+                        projectId="storybook"
+                        measures={[MEASURE_1, MEASURE_2]}
+                        rows={[ATTRIBUTE_1]}
+                        onExportReady={onExportReady}
+                        onError={onErrorHandler}
+                        LoadingComponent={null}
+                        ErrorComponent={null}
+                    />
+                )}
+            </WrapperComponent>
+        </div>
+    ))
+    .add('export visualization data', () => (
+        <div style={WRAPPER_STYLE}>
+            <WrapperComponent>
+                {(onExportReady: OnExportReady) => (
+                    <Visualization
+                        projectId="storybook"
+                        uri={'/gdc/md/storybook/obj/1003'}
+                        onExportReady={onExportReady}
+                        onError={onErrorHandler}
+                        locale="en-US"
+                        LoadingComponent={null}
+                        ErrorComponent={null}
+                    />
+                )}
+            </WrapperComponent>
+        </div>
+    ))
+    .add('export data to CSV', () => (
+        <div style={WRAPPER_STYLE}>
+            <WrapperComponent>
+                {(onExportReady: OnExportReady) => (
+                    <Visualization
+                        projectId="storybook"
+                        uri={'/gdc/md/storybook/obj/1003'}
+                        onExportReady={onExportReady}
+                        onError={onErrorHandler}
+                        locale="en-US"
+                        LoadingComponent={null}
+                        ErrorComponent={null}
+                    />
+                )}
+            </WrapperComponent>
+        </div>
+    ))
+    .add('export data with custom name', () => (
```

Comment:
what is a difference between these two last scenarios and '.add('export table data', () => ('?

Reply:
My mistake. They are the same